tests/Fixtures/inc/classes/optimization/CSS/Combine_Google_Fonts/optimize.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// phpcs:disable WordPress.WP.EnqueuedResources.NonEnqueuedStylesheet
+
+return [
+
```

Comment:
@crystinutzaa Let's add one more test data set that doesn't have the ending `|` character, i.e. just to make sure it runs with and without it.